services/npm/npm-unpacked-size.service.js

```diff
@@ -0,0 +1,65 @@
+import Joi from 'joi'
+import prettyBytes from 'pretty-bytes'
+import { pathParam } from '../index.js'
+import NpmBase, { packageNameDescription } from './npm-base.js'
+
+export default class NpmUnpackedSize extends NpmBase {
+  static category = 'size'
+
+  static route = {
+    base: 'npm/unpacked-size',
+    pattern: ':packageName/:version*',
```

Comment:
The badge should accept a scoped or unscoped package name. Can you

- Fix the route + code (route should be `scope(@[^/]+)?/:packageName/:version*` )
- Add service tests for scoped package with and without version